gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-version = 1.3.6
+version = 1.3.6-paf1
```

Comment:
It should not be committed

Reply:
Absolutely.
That should have been on some other branch.
Please just consider using 1 line change, just yourself?